docs/user/openstack/ovs-dpdk.md

```diff
@@ -30,9 +30,9 @@ This may be accomplished in two ways:
     ``` yaml
     platform:
       openstack:
-         clusterOSImageProperties:
-            - hw_cpu_policy: dedicated
-            - hw_mem_page_size: large
+        clusterOSImageProperties:
+          hw_cpu_policy: dedicated
+          hw_mem_page_size: large
```

Comment:
There is another place:

https://github.com/openshift/installer/blob/e5ae366c5505ac37577c53e972048cd22602977a/docs/user/openstack/ovs-dpdk.md?plain=1#L70-L78